### DISCUSSION TITLE
Romanian version is not working - see the video

### DISCUSSION THREAD

**Discussion by diaovidiu** [tim-hub/obsidian-bible-reference#220] [Q&A] [answered]
Hi everyone,

I would very much appreciate if you could help me understand and fix, the problem I have when trying to use the Romanian Bible version.

When I do set as the Bible version the Romanian one, calling out the Bible using -- or ++ is not working. However, if/when switching to English (for example) is all good.

Here's a video of the tested scenario: https://monosnap.com/file/x6gQ5zxGQXFYDyalPn68fWWSHp1nE1

thank you!

**Comment by tim-hub** [comment]
I couldn't see the video on my phone somehow,

Can you share which translation of book name do you use?

**Reply by tim-hub** [reply]
Hi @diaovidiu 

I checked your video, the Romanian book names are supported at current stage. 

So far we have only supported these translations of book names.

https://github.com/Antioch-Tech/bible-reference-toolkit/tree/master/packages/bible-book-names-intl/src/data/translations

**Reply by Nicolai9852** [reply]
Hey @tim-hub

I'll happily provide the Danish abbreviations to that repo, if you would like that? And if it's still in use?

**Reply by tim-hub** [reply]
> Hey @tim-hub
> 
> I'll happily provide the Danish abbreviations to that repo, if you would like that? And if it's still in use?

If you're have it handy, yes, really appreciate it.

**Reply by Nicolai9852** [reply]
Dear @tim-hub 

I believe, you should have recieved a pull request now regarding it.
Please note the two comments, I have made in it.

Edit: Here is the link https://github.com/Antioch-Tech/bible-reference-toolkit/pull/16

**Comment by diaovidiu** [comment]
Hi Tim, thanks for taking the time to reply!
Based on the en.json model I have updated the ro.json. 
Now, how can I use it now?
thanks, Tim

[ro.json](https://github.com/user-attachments/files/16622517/ro.json)

**Comment by diaovidiu** [comment]
So, can anyone provide instructions on how to use the above JSON I created for the Romanian book name translations? thanks

**Reply by tim-hub** [reply, marked as answer]
This is planned to be included in the next release

**Comment by diaovidiu** [comment]
Excellent job! It's working. Thank you!
